servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -128,29 +129,88 @@ def get_job(id, **kwargs):
 
 
 def format_task(task_name, task_metadata):
+    # check to see if task is scattered
+    is_scattered_task = len(
+        set([item.get('shardIndex') for item in task_metadata])) > 1
```

Comment:
This should probably be more like "is there a `shardIndex` that isn't -1?"

It's possible that this wouldn't work properly on a scatter that's got a width of 1